crates/src/grpc/mod.rs

```diff
@@ -7,7 +7,10 @@ use solana_sdk::{
 };
 pub mod grpc_subscriber;
 use grpc_subscriber::{AccountFilter, GrpcConnectionOpts};
+use yellowstone_grpc_proto::prelude::{Transaction, TransactionStatusMeta};
 
+/// grpc transaction update callback
+pub type OnTransactionFn = dyn Fn(TransactionUpdate) + Send + Sync + 'static;
```

Comment:
this could pass by ref by default and the event subscriber can just copy if it needs to